CONTRIBUTING.md

```diff
@@ -18,22 +18,25 @@ Project governance is documented [here](https://github.com/open-feature/communit
+Adherence to these guidelines is required for 1.0 artifact releases, to the satisfaction of the [Technical Steering Committee](https://github.com/open-feature/community/blob/main/governance-charter.md#technical-steering-committee-tsc).
+
+| Requirement                     | Recommended solution(s)                                                                                                                   | Notes                                                       |
+| ------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------- | ----------------------------------------------------------- |
+| automated publishing            | github actions (with permissions applying principle of least privilege), language-specific tools (Maven, nuget, NPM, etc)                 | required                                                    |
+| container, base image scanning  | [snyk][snyk], [trivy][trivy]                                                                                                              | required                                                    |
+| code ownership                  | branch protection rules\* and CODEOWNERS files                                                                                            | required                                                    |
+| dependency analysis             | [snyk][snyk]                                                                                                                              | required                                                    |
+| dependency auto-updates         | [Renovate][renovate]\*\*, [Dependabot][dependabot]                                                                                        | required                                                    |
+| image signing                   | language-specific tools, [cosign][cosign]                                                                                                 | required                                                    |
+| semantic versioning, changelogs | [Semantic Versioning][semantic-versioning], [Conventional Commits][conventional-commits], [Release Please][release-please]\*\*\*          | required                                                    |
+| unit, integration testing       | github actions (with permissions applying principle of least privilege), language-specific tools (JUnit, Jest, etc), [Cucumber][cucumber] | required, with coverage metrics up to maintainer discretion |
+| artifact signing                | language-specific tools, [cosign][cosign]                                                                                                 | recommended, where supported by language-specific tools     |
```

Comment:
I'd like to make this one required, but it's difficult, and simply not supported by a lot of package managers.